README.md

```diff
@@ -116,7 +116,7 @@ Get the id so we can use it in the next step.
 
     $ id -u workload
 
-Call the registration API with *spire-server register* providing the workload user id.
+Call the entry create API with *spire-server entry create* providing the workload user id.
```

Comment:
nit: the underlying API is still referred to as the `registration API`. We can keep the `Call the registration API` text, or maybe we should update the line entirely to read more like `Create a new registration entry with *spire-server entry create*` or something like that?